docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst

```diff
@@ -0,0 +1,71 @@
+.. _pilotsWithTokens:
+
+=====================================
+Submitting pilots to CEs using tokens
+=====================================
+
+
+This guide outlines the process of setting up DIRAC to submit pilots using access tokens obtained via a `client_credentials` flow from a token provider.
+
+Setting up an `IdProvider`
+--------------------------
+
+- Set up an OAuth2 client in the token provider and obtain a `client_id` and a `client_secret`.
+
+    .. warning:: The client credentials obtained are confidential, store them in a secure place.
+       Any malicious user able to get access to them would be able to generate access tokens on your behalf.
+       To avoid any major issue, we recommend you to only grant essential privileges to the client (`compute` scopes).
+
+- Add the client credentials in your local `dirac.cfg` configuration such as::
```

Comment:
```suggestion
- Add the client credentials in your local `dirac.cfg` configuration such as:
```
(Maybe also too much indent for the next block?)

Please have a look at https://dirac--7176.org.readthedocs.build/en/7176/AdministratorGuide/HowTo/pilotsWithTokens.html